core/src/main/java/com/google/cloud/sql/IpType.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.sql;
+
+/** IP types that for connecting to a Cloud SQL Database. */
+public enum IpType {
+  PUBLIC("PRIMARY"),
```

Comment:
PRIMARY is a value returned by the Cloud SQL Admin API, but not part of this project's public API. So we should either switch this to match our public API ("PIUBLIC"), or remove the strings entirely.

Reply:
I see that this code base hasn't been strict about that in the past. We need to anchor on PUBLIC to match our public API and then transform it in the SQL Admin API fetcher (where public maps to primary).